### PR TITLE
Switch to new (ASF) license

### PR DESCRIPTION
**Pull request by levitte**
-----

This is a takeover of #45, on request

### CODE REVIEW

**Comment by richsalz** [comment]
Looks good to me.  Thanks.

**Comment by levitte** [comment]
Cool.  This needs another @openssl/omc approval, though.

After this and openssl/openssl#7724 are in, I can start pushing the license change in the main source.

**Review by paulidale — APPROVED** [review]
LGTM

File: source/license.html [hunk 1]
```diff
@@ -11,16 +11,30 @@
...
+            For the 3.0.0 release, and later releases derived from that,
+            <a href="apache-license-2.0.txt">the Apache License v2</a>
+            applies.
+            This also applies to the git "master" branch.
+            </p>
+            <p>
+            For any release made before OpenSSL 3.0.0 (namely
+            the 1.1.1, 1.1.0, 1.0.2, and all prior releases including
+            those not currently supported),
+            <a href="license-ssleay.txt">the OpenSSL license</a>
```
Comment (mattcaswell) [inline source/license.html]: Technically both the OpenSSL license *and* the SSLeay license applies.

**Review by mattcaswell — CHANGES_REQUESTED** [review]

File: source/license.html (hunk 1 above)
Comment (levitte) [reply source/license.html]: I can fix that

**Review by mattcaswell — APPROVED** [review]

**Comment by levitte** [comment]
Merged.

eb36cdbb681ca71c469661f16ed1bd28207754c6 Switch to new (ASF) license

**Comment by levitte** [comment]
Now merged for real.

ad8f7120bad64bcc43861c36eedcf29fc2728f13 Switch to new (ASF) license

**Pull request closed without merging**